library/src/main/java/com/pokegoapi/api/inventory/Inventories.java

```diff
@@ -139,6 +139,12 @@ public void updateInventories(GetInventoryResponse response) {
 
 		for (InventoryItemOuterClass.InventoryItem inventoryItem
 				: response.getInventoryDelta().getInventoryItemsList()) {
+			
+			// Remove released Pokemon from bag.
+			if(inventoryItem.getDeletedItem().getPokemonId() != 0) {
```

Comment:
[ant:checkstyle] [WARN] /home/jasper/Documents/PokemonGo/PokeGOAPI-Java/library/src/main/java/com/pokegoapi/api/inventory/Inventories.java:144:9: WhitespaceAround: 'if' is not followed by whitespace. Empty blocks may only be represented as {} when not part of a multi-block statement (4.1.3) [WhitespaceAround]